### DISCUSSION TITLE
ContextFlyout only support pointer placement?

### DISCUSSION THREAD

**Discussion by Sunzhuokai** [AvaloniaUI/Avalonia#17520] [Q&A] [answered]
https://github.com/AvaloniaUI/Avalonia/blob/331d73efa34989a70c8bb47aeaf4536cb44f0ad3/src/Avalonia.Controls/Flyouts/PopupFlyoutBase.cs#L502

**Comment by maxkatz6** [comment, marked as answer]
When it's triggered by a right click, IIRC, that's the behavior. Yes.
If context menu is triggered by code or via keyboard, then it will use Placement property.
